src/guides/v2.2/cloud/live/live.md

```diff
@@ -37,33 +33,65 @@ We strongly recommend testing in these environments due to the complexity of you
 
-Each site to be scanned must be registered through Magento Security Scan tab. This registration process includes acceptance of Magento’s disclaimer prior to scanning. You control both scan scheduling and the authorization of personnel to be notified when each scan is completed. Scans can be scheduled for either a specific, recurring date and time or on-demand as required.
+The Magento Security Scan Tool uses several user agent strings to simulate real-life malware activity. You might see the following user agents in your analytics or access logs:
 
+```text
+Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0
+GuzzleHttp/6.3.3 curl/7.29.0 PHP/7.1.18
+Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.120 Safari/537.36
+Visbot/2.0 (+http://www.visvo.com/en/webmasters.jsp;bot@visvo.com)
+```
+
+## Scan your site
+
+{:.procedure}
 To scan your site:
 
 1. Access your [Magento Marketplace account](https://account.magento.com/customer/account/login).
 1. Click the Security Scan tab and select **Go to Security Scan**.
-1. In the **Actions** column for the site, select Run Scan. A notification status displays the scheduled scan.
+1. In the _Actions_ column for the site, select **Run Scan**. A notification status displays the scheduled scan.
 
+{:.procedure}
 To review the report:
 
 1. When the report completes, a notification displays.
 1. In the site row, select the report you want to view from the **Reports** column. The order is latest to oldest.
 
 The report lists issues including Failed Scans, Unidentified Results, and Successful Scans. Each entry provides detailed information for the scan, a list of issues to investigate, and actions to take. Some of these actions may require downloading and installing security patches. You can add those to a development branch on your local workstation.
 
-Scan results include a general label that describes whether a site passed or failed plus detailed information about the checks performed. Failed indicates that the website contains a serious vulnerability, while unidentified suggests that a deeper review is required by your team or hosting provider to determine if further action is required. We also provide suggested remediation steps for each failed security test. Security scan results are protected and viewable only by the registered user, and notifications of scan completion are restricted to the users designated in the site registration process.
+Scan results include a label that describes scan pass or fail status with detailed information about the checks performed:
+
+*  "Failed" indicates that the website contains a serious vulnerability.
```

Comment:
I would avoid the word "serious" since, e.g. SSL TLS check has a minor "weight" and not SO serious... Also the failure is possible on the multiple reasons in addition to found vulnerability: undetected patch, malware found, server misconfiguration, vulnerable extension found.
Maybe instead of "...website contains a serious vulnerability..." we should put something like "... scanner fund a known vulnerability that should be reviewed and addressed accordingly..."